Local node doesn't work with mirror node version > `0.110.0-rc1`

### Description

There is a issue with the mirror-node-importer, and that's the log from it
```
SQL State  : 42704
Error Code : 0
Message    : ERROR: operator class "gin_trgm_ops" does not exist for access method "gin"
Location   : db/migration/v1/V1.98.2__token_name_index.sql (/app/BOOT-INF/classes/db/migration/v1/V1.98.2__token_name_index.sql)
Line       : 1
Statement  : create index if not exists token__name on token using gin (name gin_trgm_ops)

	at org.flywaydb.core.internal.sqlscript.DefaultSqlScriptExecutor.handleException(DefaultSqlScriptExecutor.java:252)
	at org.flywaydb.core.internal.sqlscript.DefaultSqlScriptExecutor.executeStatement(DefaultSqlScriptExecutor.java:214)
	at org.flywaydb.core.internal.sqlscript.DefaultSqlScriptExecutor.execute(DefaultSqlScriptExecutor.java:133)
	at org.flywaydb.core.internal.resolver.sql.SqlMigrationExecutor.executeOnce(SqlMigrationExecutor.java:65)
	at org.flywaydb.core.internal.resolver.sql.SqlMigrationExecutor.lambda$execute$0(SqlMigrationExecutor.java:57)
	at org.flywaydb.core.internal.database.DefaultExecutionStrategy.execute(DefaultExecutionStrategy.java:27)
	at org.flywaydb.core.internal.resolver.sql.SqlMigrationExecutor.execute(SqlMigrationExecutor.java:56)
	at org.flywaydb.core.internal.command.DbMigrate.doMigrateGroup(DbMigrate.java:374)
	... 41 common frames omitted
Caused by: org.postgresql.util.PSQLException: ERROR: operator class "gin_trgm_ops" does not exist for access method "gin"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2725)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2412)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:371)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:502)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:419)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:341)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:326)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:302)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:297)
	at org.flywaydb.core.internal.jdbc.JdbcTemplate.executeStatement(JdbcTemplate.java:210)
	at org.flywaydb.core.internal.sqlscript.ParsedSqlStatement.execute(ParsedSqlStatement.java:89)
	at org.flywaydb.core.internal.sqlscript.DefaultSqlScriptExecutor.executeStatement(DefaultSqlScriptExecutor.java:206)
```

### Steps to reproduce

Update local.json mirror-node version to `0.110.0-rc1`, run the node, and try to fetch blocks from the mirror-node-rest endpoint
[http://localhost:5551/api/v1/blocks?order=desc](http://localhost:5551/api/v1/blocks?order=desc)
```
{"key": "MIRROR_IMAGE_TAG", "value": "0.110.0-rc1"},
```
### Solution
copy [this](https://github.com/hashgraph/hedera-mirror-node/blob/main/hedera-mirror-importer/src/main/resources/db/scripts/init.sh) init.sh, into [this one in the local node](https://github.com/hashgraph/hedera-local-node/blob/main/compose-network/mirror-node/init.sh)


### Additional context

_No response_

### Hedera network

_No response_

### Version

latest

### Operating system

None

--- a/.env
+++ b/.env
@@ -4,8 +4,8 @@ NETWORK_NODE_IMAGE_PREFIX=gcr.io/hedera-registry/
 NETWORK_NODE_IMAGE_NAME=main-network-node
 
 #### Image Tags/Hashes ####
-NETWORK_NODE_IMAGE_TAG=0.52.0-alpha.4
-HAVEGED_IMAGE_TAG=0.52.0-alpha.4
+NETWORK_NODE_IMAGE_TAG=0.52.0-alpha.6
+HAVEGED_IMAGE_TAG=0.52.0-alpha.6
 
 #### Java Process Settings ####
 PLATFORM_JAVA_HEAP_MIN=256m
@@ -29,7 +29,7 @@ PYTHON_VERSION=python3.7
 
 #### MirrorNode Prefixes & Tags ####
 MIRROR_IMAGE_PREFIX=gcr.io/mirrornode/
-MIRROR_IMAGE_TAG=0.108.0
+MIRROR_IMAGE_TAG=0.110.0-rc1
 
 #### MirrorNode settings ####
 MIRROR_POSTGRES_IMAGE=postgres:14-alpine
@@ -45,7 +45,7 @@ MINIO_ROOT_PASSWORD=minioadmin
 
 #### JSON RPC Relay Prefixes & Tags ####
 RELAY_IMAGE_PREFIX=ghcr.io/hashgraph/
-RELAY_IMAGE_TAG=0.51.1
+RELAY_IMAGE_TAG=0.52.1
 
 #### JSON RPC Relay limits ####
 RELAY_MEM_LIMIT=768m

--- a/.github/workflows/flow-pull-request-checks.yaml
+++ b/.github/workflows/flow-pull-request-checks.yaml
@@ -103,7 +103,7 @@ jobs:
           npm ci
 
       - name: Start local node
-        run: npm run start -- -d
+        run: npm run start -- -d --verbose=trace
 
       - name: Prepare and run the tests
         run: |

--- a/compose-network/mirror-node/application.yml
+++ b/compose-network/mirror-node/application.yml
@@ -11,6 +11,7 @@ hedera:
             persist:
               transactionBytes: false
               transactionRecordBytes: false
+              transactionHash: true
             redis:
               enabled: false
           sidecar:

--- a/compose-network/mirror-node/init.sh
+++ b/compose-network/mirror-node/init.sh
@@ -6,7 +6,8 @@ export PGDATABASE="${POSTGRES_DB:-postgres}"
 export PGHOST="${PGHOST}"
 export PGUSER="${POSTGRES_USER:-postgres}"
 
-DB_SPECIFIC_EXTENSION_SQL="create extension btree_gist;"
+DB_SPECIFIC_EXTENSION_SQL="create extension btree_gist;
+                           create extension pg_trgm;"
 # PostgreSQL 16 requires role A to have Admin privilege on role B in order to grant role B to any other role, and there
 # are several versioned v1 migrations that grant :importerUsername to :ownerUsername.
 DB_SPECIFIC_IMPORTER_ROLE_ADMIN="admin :ownerUsername"
@@ -15,6 +16,7 @@ DB_SPECIFIC_SQL="alter user :ownerUsername with createrole;"
 # v2 schema no longer creates the REST API user, while v1 schema still does
 if [[ "${SCHEMA_V2}" == "true" ]]; then
   DB_SPECIFIC_EXTENSION_SQL="create extension if not exists btree_gist;
+                             create extension if not exists pg_trgm;
                              create extension citus;
                              grant all privileges on database :dbName to :ownerUsername;
                              grant create on database :dbName to :ownerUsername;

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@hashgraph/hedera-local",
-  "version": "2.27.1",
+  "version": "2.28.0",
   "description": "Developer tooling for running Local Hedera Network (Consensus + Mirror Nodes).",
   "main": "index.ts",
   "scripts": {
@@ -56,12 +56,12 @@
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-prettier": "^5.1.3",
     "husky": "^9.0.11",
-    "mocha": "^10.6.0",
+    "mocha": "^10.7.0",
     "nyc": "^17.0.0",
     "prettier": "^3.3.2",
     "semantic-release": "^24.0.0",
     "sinon": "^18.0.0",
-    "typescript": "^5.5.3"
+    "typescript": "^5.5.4"
   },
   "dependencies": {
     "@hashgraph/sdk": "^2.48.1",
@@ -71,10 +71,10 @@
     "detect-port": "^1.6.1",
     "dockerode": "^4.0.2",
     "dotenv": "^16.4.5",
-    "ethers": "^6.13.1",
+    "ethers": "^6.13.2",
     "js-yaml": "^4.1.0",
-    "rimraf": "^6.0.0",
-    "semver": "^7.6.2",
+    "rimraf": "^6.0.1",
+    "semver": "^7.6.3",
     "shelljs": "^0.8.5",
     "ts-mocha": "^10.0.0",
     "yargs": "^17.7.2"

--- a/src/configuration/local.json
+++ b/src/configuration/local.json
@@ -1,9 +1,9 @@
 {
   "imageTagConfiguration": [
-    {"key": "NETWORK_NODE_IMAGE_TAG", "value": "0.52.0-alpha.5"},
-    {"key": "HAVEGED_IMAGE_TAG", "value": "0.52.0-alpha.5"},
-    {"key": "MIRROR_IMAGE_TAG", "value": "0.108.0"},
-    {"key": "RELAY_IMAGE_TAG", "value": "0.51.1"},
+    {"key": "NETWORK_NODE_IMAGE_TAG", "value": "0.52.0-alpha.6"},
+    {"key": "HAVEGED_IMAGE_TAG", "value": "0.52.0-alpha.6"},
+    {"key": "MIRROR_IMAGE_TAG", "value": "0.110.0-rc1"},
+    {"key": "RELAY_IMAGE_TAG", "value": "0.52.1"},
     {"key": "MIRROR_NODE_EXPLORER_IMAGE_TAG", "value": "24.4.0"}
   ],
   "envConfiguration": [

